configuration.go

```diff
@@ -52,6 +56,18 @@ func (s *Source) HandlePropertySources(handler PropertySourceHandler) {
 	}
 }
 
+func (s *Source) Get(key string, defaultValue string) interface{} {
+	for _, propertySource := range s.PropertySources {
+		for _key, value := range propertySource.Source {
```

Comment:
rename `_key` to `sourceKey` to make it more clear. I was confused at first what `_key` was